site/templates/film/item.jet

```diff
@@ -33,13 +33,13 @@
           {{if config("default_image_type") == "portrait"}}
             {{yield sponsor() film}}
 
-            {{if film.ImageMap["Portrait"] != ""}}
               <div class="poster poster-portrait">
-                <s72-availability-status data-slug="{{film.Slug}}"></s72-availability-status>
-                <s72-image src="{{film.ImageMap["Classification"]}}" alt="" class="classification-image">
-                <s72-image src="{{film.ImageMap["Portrait"]}}" alt="{{film.Title}}" class="poster poster-image poster-image-portrait"></s72-image>
+                {{if film.ImageMap["Portrait"] != ""}}
```

Comment:
Is the purpose of this change to use that poster div to force the other content to the right a bit?  You could totally do it by changing the `auto` column width [here](https://github.com/shift72/core-template/blob/b359371cca68a2a1f11f2919898c8272cffb6cb0/site/styles/_meta-detail.scss#L113) to the desired image width... then we don't need that potentially empty poster container floating around.

Reply:
> Is the purpose of this change to use that poster div to force the other content to the right a bit?

Nah its to occupy the empty space when the image isn't present.

> You could totally do it by changing the `auto` column width [here](https://github.com/shift72/core-template/blob/b359371cca68a2a1f11f2919898c8272cffb6cb0/site/styles/_meta-detail.scss#L113) to the desired image width... then we don't need that potentially empty poster container floating around.

I tried this out to see what you mean, I didn't like how it specified a width in two places (and didn't want to remove the poster width)

Reply:
> Nah its to occupy the empty space when the image isn't present.

That's what I meant by force the other stuff to the right lol.  You could add a css var for the poster widths or maybe just have width on the column and have the poster be 100% wide?